org.uqbar.project.wollok.lib/src/wollok.wlk

```diff
@@ -357,7 +382,14 @@ package lang {
 			acc
 		})
 
-		method filter(closure) = self.fold(self.newInstance(), { acc, e =>
+		/**
+		 * Returns a new collection that contains the elements that meet a given condition.
+		 * The condition is a closure argument that takes a single element and returns a boolean.
+		 * @returns another collection (same type as self one)
```

Comment:
I tend to use `element` for collection members, `argument` or `parameter` - depending context - for closure inputs and `components` for tuple members.